internal/db/nosql/cassandra/option.go

```diff
@@ -125,23 +140,33 @@ func WithConnectTimeout(dur string) Option {
 	}
 }
 
+// WithPort returns the option to set the port number
 func WithPort(port int) Option {
 	return func(c *client) error {
+		if port <= 0 || port > math.MaxUint16 {
```

Comment:
note: we don't need to cast the uint to int before doing comparison
https://github.com/vdaas/vald/blob/45b66c37b96f517b1fa193308fdde7eddcc111ab/internal/db/nosql/cassandra/option_test.go#L614